README.md

```diff
@@ -39,7 +39,79 @@ In your application.js:
 ```
 ## Usage
 
-We're not sure yet.
+Given a simple application with Tasks belonging to Projects, the first thing to do is allow your Project model to accepts_nested_attributes_for.
+
+**Models**
+
+```ruby
+class Project < ApplicationRecord
```

Comment:
add a comment here
` # models/project.rb`

Reply:
seems redundant, I don't thinks its needed considering Cocoon doesn't do it.

Reply:
I did it though.